src/routes/api/create-publication.js

```diff
@@ -1,19 +1,20 @@
 import got from "got";
+import { resourceLimits } from "worker_threads";
```

Comment:
Is this a VSCode auto import? (VSCode is sometimes a bit dumb about auto importing) I ask because it doesn't seem to used anywhere. 🙂

Reply:
oops. Yes, I really need to turn off the auto importing. It is never useful.

Reply:
Same 😄